examples/streams-join/streams-join.bal

```diff
@@ -2,38 +2,38 @@ import ballerina/http;
 import ballerina/mime;
 import ballerina/io;
 
-
+// Create an object type named `ProductMaterial` and define the attributes.
 type ProductMaterial {
     string name;
     float amount;
 };
 
+// Create an object type named `MaterialUsage` and define the attributes.
```

Comment:
Shall we remove this comment? It states the obvious. If we are commenting, we should say why, not what.